src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/GeoserverUrlIndex.java

```diff
@@ -30,12 +19,17 @@
 import org.opengis.filter.sort.SortBy;
 import org.opengis.filter.sort.SortOrder;
 
-public class UrlIndexFeatureSource implements UrlIndexInterface {
-    static Logger LOGGER = Logging.getLogger(UrlIndexFeatureSource.class);
+import java.io.IOException;
+import java.util.*;
```

Comment:
Don't normally include whole packages - normally specify classes imported.  IDE's can normally fix this up for you (I have an organise imports option which removes unused imports, sorts the imports and so on)